Bobby Septianto

bobbyseptianto opened the pull request:
E-Commerce Server
1. TDD Server
2. Controller
3. Routing REST API User, Product, Category
4. Error Handler

Commit:
E-Commerce Server
1. TDD Server
2. Controller
3. Routing REST API User, Product, Category
4. Error Handler

Commit:
Revisi /loginAdmin

Commit:
Update minor API Doc, ProductController, Test Jest

Commit:
Read Product By Id
1. Menambahkan routing untuk read product by id
2. Revisi test JEST before all create category supaya dapat category id
3. API DOC untuk read product by id
4. Controller readProductById

Commit:
Deploy Server

bobbyseptianto commented:
DEPLOYED SERVER

Fitur Tambahan: Dynamic Category

ENV:
PORT=3000
SALT=10
SECRET=krabbypatty

Thankyouu

Commit:
SERVER - FOR CUSTOMER
1. Sequelize Cart + Wishlist
2. Add Routing Login/Register Customer
3. Add Routing + Controller => CRUD Cart
4. Add Routing + Controller => CRD Wishlist
5. Update API Docs

Commit:
Update Server - Customer
1. Revisi API Doc Error Respon Add to Cart & Add to Wishlist
2. Cart Controller:
   * Update Add to Cart cek quantity > stock ?
   * Update Read Cart with floating value total of cart
   * Update update Cart cek quantity ? stock ?
3. Add helpers money formater
4. Add column status "checkout" true/false on Carts

Commit:
Update Server - Customer
1. Revisi addToCart check quantity
2. Revisi authentication admin & customer
3. Pindah auth di dalam masing" router

Commit:
Update API Docs readProduct and readCategories for Customer

Commit:
Add Checkout Cart Routing
1. Update API Docs for Checkout "PUT"
2. Checkout ->CartController
3. Checkout Routing

Commit:
PREAPERING DEPLOY FOR UPDATED CUSTOMER SERVER

ghost commented:
DEPLOYED SERVER + ROUTING CUSTOMER

Fitur Tambahan: Dynamic Category, CRUD Wishlist, Checkout Update Stock DB & Kosongkan Cart

ENV:
PORT=3000
SALT=10
SECRET=krabbypatty

ADMIN https://ikeha-ecommerce-bob.web.app
CUSTOMER https://ikeha-homeliving.web.app

Thankyouu

Commit:
Add Increment and Decrement Routing
1. Udate API Docs Increment and Decrement
2. CartController -> add method increment and decrement
3. Routing /carts/:id/incrementQuantity & /decrementQuantity

Commit:
Revisi Logic Checkout pake PromiseAll
